examples/abitest/module_0/rust/src/proto/mod.rs

```diff
@@ -1,2 +1,5 @@
+//! Auto-generated code for processing protocol buffer message definitions and
+//! gRPC service definitions.
```

Comment:
Is this code __actually__ auto-generated? The other day @ipetr0v and I spent a long time trying to figure it out, and we arrived to the conclusion that it is not, or that we don't understand what is generating it. @daviddrysdale could you help us solve the mystery?

Reply:
Hmm, I think you're right.  I guess the comment means that the sub-modules listed below are auto-generated, not that this index of them is.

I've moved the various `mod.rs` files to have a license header.

Reply:
:100: this is valuable IP